README.md

```diff
@@ -115,7 +115,7 @@ None.
 - name: "Install Minio"
   hosts: all
   roles:
-    - atosatto.minio
+    - { role: atosatto.minio, become: yes }
```

Comment:
Normally I would just put this at the playbook level, rather than at the role level.

Reply:
I have no stake. :)

Reply:
@SuperQ thoughts about merging this example, or no?

Reply:
@till would you mind changing it to something like

```
- name: "Install Minio"
  hosts: all
  become: yes
  roles:
    - { role: atosatto.minio }
```

Reply:
@atosatto Of course not! (Done ✅)